Libraries/src/Amazon.Lambda.Annotations.nuspec

```diff
@@ -0,0 +1,25 @@
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>Amazon.Lambda.Annotations</id>
+        <version>0.1.0-preview1</version>
+        <authors>Amazon Web Services</authors>
+        <tags>AWS Amazon Lambda</tags>
+        <description>Annotations that can be added to Lambda projects to generate C# code and CloudFormation templates.</description>
```

Comment:
@ik130 need your input here, can you provider a description that we want to use on nuget.org.